### ISSUE TITLE
Status command error

### ISSUE THREAD

**Issue by j-broo** [Salamek/netkeeper#2]
I picked up something else:

```
root@J-Pi:/home/pi# netkeeper status
Loading config from /etc/netkeeper/config.yml
+-----------------------------------+
| Ping Info                         |
| --------------- + --------------- |
| Item            | Value           |
| --------------- + --------------- |
| 208.67.220.220  | ONLINE: 0.0288s |
| 9.9.9.9         | ONLINE: 0.0304s |
| 172.217.170.110 | ONLINE: 0.0308s |
+-----------------------------------+
Traceback (most recent call last):
  File "/usr/bin/netkeeper", line 33, in <module>
    sys.exit(load_entry_point('netkeeper==2.0.3', 'console_scripts', 'netkeeper')())
  File "/usr/lib/python3/dist-packages/netkeeper/__main__.py", line 6, in main
    _main()
  File "/usr/lib/python3/dist-packages/netkeeper/bin/netkeeper.py", line 416, in main
    getattr(command, 'chosen')()  # Execute the function specified by the user.
  File "/usr/lib/python3/dist-packages/netkeeper/bin/netkeeper.py", line 406, in status
    'WAN IP': monitoring['WanIPAddress'],
KeyError: 'WanIPAddress'
root@J-Pi:/home/pi#
```

**Comment by Salamek** [comment]
Hmm looks like your router is not returning all expected values (that my router is providing), so i have modified the code to not crash when expected value is not found... just update to 2.0.5 ...

**Comment by j-broo** [comment]
Thanks, yes that fixed it. FYI I am using a Huawei H112-372. The WAN IP now shows as "???".

**Comment by j-broo** [comment]
Are you perhaps interrogating the IPv6 WAN address?

Here are my values from the router's web UI:
`
WAN IP Address: 10.213.106.xx
WAN IPv6 Address: Unknown
`
I have IPV6 disabled in my APN settings.

Also, "netkeeper status" shows 0/5 bars for signal but the router UI shows 3/5. No big deal but letting you know. Happy to test for you if needed.

**Comment by Salamek** [comment]
I'm using this ~code to get this^ info:

```python
monitoring = client.monitoring.status()

rows = {
   'Signal': '{}/{}'.format(monitoring.get('SignalIcon'), monitoring.get('maxsignal')),
   'WAN IP': monitoring.get('WanIPAddress', '???'),
}

```

So could you use this code so we can see what data your router actually provides(change URL, username, password to match your config)


test.py
```python
#!/usr/bin/env python3


from huawei_lte_api.Connection import Connection
from huawei_lte_api.Client import Client


with Connection('ROUTER_URL', username='admin', password='password') as connection:
    client = Client(connection)
    print(client.monitoring.status())
```

Run it using

```bash
python3 test.py
```

Censor any information you deem private and post the output here... thx

**Comment by j-broo** [comment]
Here you go:

`{'ConnectionStatus': '901', 'WifiConnectionStatus': None, 'SignalStrength': None, 'SignalIcon': '0', 'CurrentNetworkType': '20', 'CurrentServiceDomain': '2', 'SignalIconNr': '3', 'EndcStatus': '0', 'endcRestrictedStatus': '0', 'RoamingStatus': '0', 'BatteryStatus': None, 'BatteryLevel': None, 'BatteryPercent': None, 'simlockStatus': '0', 'PrimaryDns': '41.216.200.5', 'SecondaryDns': '41.216.200.6', 'wififrequence': '1', 'flymode': '0', 'PrimaryIPv6Dns': None, 'SecondaryIPv6Dns': None, 'CurrentWifiUser': '3', 'TotalWifiUser': '64', 'currenttotalwifiuser': '64', 'ServiceStatus': '2', 'SimStatus': '1', 'WifiStatus': '1', 'CurrentNetworkTypeEx': '111', 'maxsignal': '5', 'wifiindooronly': '0', 'cellroam': '1', 'classify': 'cpe', 'usbup': '0', 'wifiswitchstatus': '1', 'WifiStatusExCustom': '0', 'hvdcp_online': '0', 'speedLimitStatus': '0'}`

**Comment by Salamek** [comment]
Ok so wan address i missing and looks like `SignalIconNr` is correct value for your signa?

**Comment by Salamek** [comment]
and can you try to run this?

```python
#!/usr/bin/env python3


from huawei_lte_api.Connection import Connection
from huawei_lte_api.Client import Client


with Connection('ROUTER_URL', username='admin', password='password') as connection:
    client = Client(connection)
    print(client.device.information())
```

**Comment by j-broo** [comment]
> Ok so wan address i missing and looks like `SignalIconNr` is correct value for your signa?

Yes I see WAN is missing. `SignalIconNr` seems to be the "bars".

**Comment by j-broo** [comment]
> and can you try to run this?

Results:
`
{'DeviceName': 'H112-372', 'SerialNumber': 'LUNDW20603xxxxx', 'Imei': '8607300409xxxxx', 'Imsi': '6557300006xxxxx', 'Iccid': '892738000040252xxxxx', 'Msisdn': None, 'HardwareVersion': 'WL2H112SM', 'SoftwareVersion': '10.0.5.1(H351SP5C1522)', 'WebUIVersion': 'WEBUI 10.0.5.1(W2SP19C1522)', 'MacAddress1': '74:0A:E1:7F:6D:xx', 'MacAddress2': None, 'WanIPAddress': '10.213.106.xx', 'wan_dns_address': '41.216.200.5,41.216.200.6', 'WanIPv6Address': None, 'wan_ipv6_dns_address': None, 'ProductFamily': 'LTE', 'Classify': 'cpe', 'supportmode': 'LTE|WCDMA|GSM', 'workmode': 'NR-5GC', 'submask': '255.255.255.255', 'Mccmnc': '65573', 'iniversion': 'H112-372-CUST 10.0.1.1(C1522)', 'uptime': '66578', 'ImeiSvn': '01', 'WifiMacAddrWl0': '74:0A:E1:7F:6D:xx', 'WifiMacAddrWl1': '74:0A:E1:7F:6D:xx', 'spreadname_en': '5G CPE Pro', 'spreadname_zh': '5G CPE Pro'}
`

**Comment by Salamek** [comment]
There is our wan address! :-)

**Comment by j-broo** [comment]
Awesome. Huawei not always consistent it seems...

**Comment by Salamek** [comment]
should be fixed in 2.0.6
